daq/forchestrator.py

```diff
@@ -61,6 +61,12 @@ def _handle_faucet_events(self):
                 LOGGER.info('DP restart %d %s', dpid, restart_type)
                 self._faucet_states_collector.process_config_change(dpid, restart_type, timestamp)
 
+            (stack_root, graph) = self._faucet_events.as_stack_topo_change(event)
+            if stack_root is not None:
+                LOGGER.info('stack topology change root:%s graph:%s', stack_root, graph)
+                self._faucet_states_collector.process_stack_topo_change(stack_root, graph, timestamp)
```

Comment:
for all of these, timestamp should be the first argument -- can you go through and update them?  I want to nip this pattern in the bud before it gets too far (missed it in the code review before)

Reply:
I shall change it for all the calls, then?

Reply:
yes

Reply:
Hey @grafnu , is there a reason why we want timestamp to be the first argument.